### DISCUSSION TITLE
Integrating AWS IAM Authentication with Strimzi-Cluster-Operator for Kafka Connect

### DISCUSSION THREAD

**Discussion by shubhamg7** [strimzi/strimzi-kafka-operator#5435] [Q&A]
Hello! My team is looking to utilize Strimzi to setup Kafka-Connect on Kubernetes. We are hoping to integrate this with AWS MSK and utilize TLS for encryption and [IAM Access Control](https://docs.aws.amazon.com/msk/latest/developerguide/iam-access-control.html) for authentication and authorization. 

I am aware that Strimzi does not support IAM auth out of the box. Looking at some discussions in [Issue 2761](https://github.com/strimzi/strimzi-kafka-operator/issues/2761#issuecomment-609091174) and [Discussion 5239](https://github.com/strimzi/strimzi-kafka-operator/discussions/5239) it seems like a solution may be possible here by tweaking environment variables in the strimzi-cluster-operator, and it does seem like people have been successful with it.

I was wondering if there were any known solutions and/or documentation written up that demonstrates if this integration is possible and how we can go about doing it?

**Strimzi Version:** 0.21.1

**Comment by scholzj** [comment]
There is nothing AWS IAM specific in the docs. Just the generic docs about mounting secrets in Kafka Connect: https://strimzi.io/docs/operators/0.22.1/full/using.html#type-ExternalConfiguration-reference ... but if you get it working it would for example make a nice [blog post](https://strimzi.io/blog/2021/07/29/how-to-write-blog-posts-for-strimzi-blog/) or docs PR.
